frontend/src/components/sc/components/instances-table/components/state-cell/state-cell.tsx

```diff
@@ -0,0 +1,5 @@
+import { Chip } from 'components/common/common';
+
+const StateCell = (state: string): JSX.Element => <Chip>{state}</Chip>;
```

Comment:
```suggestion
const StateCell = (state: string): JSX.Element => <Chip chipColor={instanceStateToChipColor[state]}>{state}</Chip>;
```
we should have color according to the design